### PR TITLE
HW 1: Part 1, 2, and 3 

### PR DESCRIPTION
**Pull request by Lynanguyen0515**
Update my readme link, project github repo link, wilfly-elytron description, and a link to a bug.

### CODE REVIEW

File: lhn2112.md [hunk 1]
```diff
@@ -6,9 +6,22 @@ UNI: lhn2112
 
 Github: [link](https://github.com/Lynanguyen0515)
 
+Readme: [link](https://github.com/Lynanguyen0515/Lynanguyen0515)
```
Comment (timkpaine) [inline lhn2112.md:9]: provide direct link to file

**Review by timkpaine — CHANGES_REQUESTED** [review]
Readme looks good, just need to make markdown files for your project and 1 of the projects to which you want to contribute

**Comment by Lynanguyen0515** [comment]
Updated Links to .md files for my project and the project I want to contribute to.

File: lhn2112.md [hunk 2]
```diff
@@ -6,9 +6,23 @@ UNI: lhn2112
 
 Github: [link](https://github.com/Lynanguyen0515)
 
+Readme: [link](https://github.com/Lynanguyen0515/Lynanguyen0515/blob/main/README.md)
 
 ## Project Proposal
 I'm planning to build a library in Java (or maybe Javascript) to automate the process of web pentesting. 
 
+Project github link: [link](https://github.com/Lynanguyen0515/pepe-pentest)
+Project .md File: [link](https://github.com/Lynanguyen0515/project-proposals-s2023/blob/main/pepe-pentest.md)
```
Comment (timkpaine) [inline lhn2112.md:15]: use relative path, like `./projects/python/pepe-pentest.md`

File: pepe-pentest.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# pepe-pentest
```
Comment (timkpaine) [inline pepe-pentest.md:1]: move to language folder like `projects/python/pepe-pentest.md`

File: wildfly-contribute.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+Description of the project:
```
Comment (timkpaine) [inline wildfly-contribute.md:1]: move to language folder

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Pull request closed without merging**